frontend/src/toolbar/debug/EventDebugMenu.tsx

```diff
@@ -0,0 +1,80 @@
+import clsx from 'clsx'
+import { useActions, useValues } from 'kea'
+import { AnimatedCollapsible } from 'lib/components/AnimatedCollapsible'
+import { TZLabel } from 'lib/components/TZLabel'
+import { LemonSwitch } from 'lib/lemon-ui/LemonSwitch'
+import { SimpleKeyValueList } from 'scenes/session-recordings/player/inspector/components/SimpleKeyValueList'
+
+import { eventDebugMenuLogic } from '~/toolbar/debug/eventDebugMenuLogic'
+
+import { ToolbarMenu } from '../bar/ToolbarMenu'
+
+export const EventDebugMenu = (): JSX.Element => {
+    const { events, isCollapsedEventRow, expandedEvent, showRecordingSnapshots } = useValues(eventDebugMenuLogic)
+    const { markExpanded, setShowRecordingSnapshots } = useActions(eventDebugMenuLogic)
+
+    return (
+        <ToolbarMenu>
+            <ToolbarMenu.Header>
+                <div className="flex flex-col pb-2 space-y-1">
+                    <div className="flex flex-row justify-around">
+                        <span className="text-xs">
+                            Seen {events.filter((e) => e.event !== '$snapshot').length} events.
```

Comment:
Nit. This should be in the logic. Especially as the list gets bigger and bigger.

Reply:
My inner Ben even said this to me when I was adding it 🤣 I should have listened to them